module/VuFind/src/VuFind/ILS/Driver/Folio.php

```diff
@@ -1181,6 +1185,16 @@ public function getMyHolds($patron)
                 'reqnum' => $hold->id,
                 // Title moved from item to instance in Lotus release:
                 'title' => $hold->instance->title ?? $hold->item->title ?? '',
+                'available' => in_array(
+                    $hold->status,
+                    $this->config['Holds']['available']
```

Comment:
Maybe we should provide a fallback to default values here in case the settings are missing -- e.g. `$this->config['Holds']['available'] ?? $defaultAvailableStatuses`, with $defaultAvailableStatuses defined earlier in the code somewhere.